.github/workflows/integration-tests.yml

```diff
@@ -7,8 +7,14 @@ on:
     branches: [ "main" ]
             
 jobs:
-  call-workflow:
+  test-plugin:
     uses: ./.github/workflows/test-plugin.yml
     with:
         triton-ref: '05dc28be0e72dd496300a31b99a21a5a5118f8e9' # known good commit "[CI] refactor workflows (#2504)"
         triton-shared-ref: ${{ github.ref }}
+
+  test-cpuref:
+    uses: ./.github/workflows/test-cpuref.yml
+    with:
+        triton-ref: '05dc28be0e72dd496300a31b99a21a5a5118f8e9' # known good commit "[CI] refactor workflows (#2504)"
```

Comment:
once we merge this, i think it would be a good idea to also run this on triton main so that we know of any regressions